zebedee-cms/src/main/java/com/github/onsdigital/zebedee/kafka/KafkaClientImpl.java

```diff
@@ -36,7 +36,12 @@ public KafkaClientImpl(String kafkaAddr, String topic) {
      */
     @Override
     public Future<RecordMetadata> produceContentPublished(String url, String dataType, String collectionID) {
-        ContentPublished value = new ContentPublished(url,dataType,collectionID);
+
```

Comment:
I wouldn't modify the URL in this class. This class is concerned with sending an event not the content of the event. The calling class(es) would be more appropriate.

Reply:
suggested changes implemented